pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -78,6 +79,16 @@ var _ = Describe("v1beta1 inference service controller", func() {
                "localGateway": "knative-serving/knative-local-gateway",
                "localGatewayService": "knative-local-gateway.istio-system.svc.cluster.local"
             }`,
+			"storageInitializer": `{
+								"image" : "kserve/storage-initializer:latest",
```

Comment:
is the tabulation correct?

Reply:
what do you mean?

Reply:
indentation :)
sometimes my english fails LOL

Reply:
updated